### ISSUE TITLE
Not working in Asp.Net core in Azure

### ISSUE THREAD

**Issue by rangav** [Bluegrams/YoutubeDLSharp#1]
Hi i am using the library and working locally, but not working when i publish to azure app service.

**Asp Version**
Asp core mvc 2.2
Azure app service

**Error:**
[youtube-dl ERROR] PermissionError: [WinError 5] Access is denied

**Comment by alxnull** [comment]
Hi, thanks for reporting this. I haven't really worked with ASP.Net yet, so I haven't experienced this bug. But I'll look into this as soon as possible.
This youtube-dl issue seems to be related: https://github.com/ytdl-org/youtube-dl/issues/10766.

**Comment by rangav** [comment]
i got it working in azure, thanks

**Comment by alxnull** [comment]
Great to here. Did you need to use some workaround to get it working?

**Comment by rangav** [comment]
Yeah I have to install python site extension latest and install youtube-dl using pip and use that url to point in the application

**Comment by rangav** [comment]
And **FrameRate** should be double - otherwise it is getting error for Vimeo url.

**Comment by alxnull** [comment]
> And FrameRate should be double - otherwise it is getting error for Vimeo url.

Thanks, this has been fixed in version 0.2.0.

The `YoutubeDLProcess` class now also supports the workaround suggested in https://github.com/ytdl-org/youtube-dl/issues/10766:

```csharp
ydlProc.PythonPath = "path\to\python.exe";
ydlProc.ExecutablePath = "path\to\youtube_dl\__main__.py";
```
